README.md

```diff
@@ -69,6 +69,42 @@ Get all tags for a model
 model.tags
 ```
 
+### Training a Named Entity Extractor
+
+Load a MITIE NER model into a trainer
+
+```ruby
+trainer = Mitie::Trainers::NER.new("total_word_feature_extractor.dat")
+```
+
+Create training instances to train against
+
+```ruby
+sample = Mitie::TrainingInstances::NER.new(["My", "name", "is", "Davis", "King", "and", "I", "work", "for", "MIT", "."])
```

Comment:
Let's remove last names from examples and tests (even though they're already public in MITIE).